### PR TITLE
LPS-178783 | Add test CanSendEmailViaActionStandalone

### PR DESCRIPTION
**Pull request by bronsonsoda**
https://issues.liferay.com/browse/LPS-178783

### CODE REVIEW

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 1]
```diff
@@ -739,4 +739,80 @@ definition {
...
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And Custom Object with an Action (Standalone)") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:768]: Can you split this into two tasks? One task to create the custom object and entry, one task to add a standalone action. You can use your best judgement for the wording of the steps.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 2]
```diff
@@ -739,4 +739,80 @@ definition {
...
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Notification action when entry is added",
+				active = "true",
+				inputExpression = '''isEmpty(oldValue("customField"))''',
+				notificationName = "Notification Template",
+				objectLabel = "Custom Object 178783",
+				thenAction = "Notification",
+				whenAction = "On After Add");
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:799]: You need to create a standalone action here. You can find examples in ObjectActions.testcase

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 3]
```diff
@@ -739,4 +739,80 @@ definition {
...
+		}
+
+		task ("When the entry is triggered manually") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject178783");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:806]: We need to trigger the standalone action manually here. Can you update the steps and the wording?

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 4]
```diff
@@ -739,4 +739,80 @@ definition {
...
+
+		task ("Then the email is sent immediately on Queue") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:808]: Then the email is sent

**Review by timpak — CHANGES_REQUESTED** [review]
Hey @bronsonsoda, good job with this test! Can you take a look at my comments below? Once you make the changes, you can forward this.

**Comment by bronsonsoda** [comment]
sf:local

**Comment by timpak** [comment]
I'm gonna take a look at this on Wednesday

**Comment by bronsonsoda** [comment]
I didnt type yes to push rats

**Review by timpak — APPROVED** [review]

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**